server/template_test.go

```diff
@@ -360,7 +359,7 @@ func TestPushedCommitsTemplate(t *testing.T) {
 			Repo:   &pushEventRepository,
 			Sender: &user,
 			Forced: bToP(true),
-			Commits: []github.PushEventCommit{
+			Commits: []*github.HeadCommit{
```

Comment:
`github.PushEventCommit` was removed from `github.com/google/go-github`. It's just a naming change.